biblatex-iso690.tex

```diff
@@ -411,25 +543,21 @@ \subsubsection{Creators}
 individuals, the title should be used as the first element of the reference.
 However, it is appropriate to include some relevant roles, e.g. the director:
 
-%\begin{ltxexample}
-\begin{verbatim}
+\begin{ltxexample}
   ..
-  editora     = {Welles, Orson}
-  editoratype = {Directed by}
+  editora     = {Welles, Orson},
+  editoratype = {Directed by},
```

Comment:
As above with the `@patent`. It would be nicer if things were strings. This shows ...